pkgs/tools/X11/xkeysnail/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchFromGitHub, makeWrapper, python3 }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "xkeysnail";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "mooz";
+    repo = pname;
+    rev = "bf3c93b4fe6efd42893db4e6588e5ef1c4909cfb";
+    sha256 = "0plcpb4ndzfsd5hj32m0g32swnhyph9sd759cdhhzmjvlq3j8q6p";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+  propagatedBuildInputs = with python3.pkgs; [
+    evdev
+    xlib
+    inotify-simple
+    appdirs
+  ];
+
+  doCheck = false;
+
+  postInstall = ''
+    mkdir -p "$out/share"
+    cp "./example/config.py" "$out/share/config.py"
+    cp "${./browser-emacs-binding.py}" "$out/share/config-emacs.py"
+
+    makeWrapper "$out/bin/xkeysnail" "$out/bin/xkeysnail-example" \
+      --add-flags "-q" --add-flags "$out/share/config.py"
+    makeWrapper "$out/bin/xkeysnail" "$out/bin/xkeysnail-browser" \
+      --add-flags "-q" --add-flags "$out/share/config-emacs.py"
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Keyboard remapping tool for X environment";
+    homepage = "https://github.com/mooz/xkeysnail";
+    platforms = platforms.linux;
+    license = licenses.gpl1;
```

Comment:
gpl1only or gpl1Plus?

Reply:
It is probably `gpl3Plus`. [Here](https://github.com/mooz/xkeysnail) is the project page.
```
This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.
```

Reply:
It seems license text was copied directly from GNU page. It says `at your option`, but does not specify it explicitly. Therefore I set the license as `gpu3Only`.

Reply:
It says `distributed under GPL` so I've set it to `gpl1Only`. `Pypi` and `AUR` pages also check out.

Reply:
[AUR](https://aur.archlinux.org/packages/xkeysnail) and [pypi](https://pypi.org/project/xkeysnail/) pages also lists as GPL.